Header header

float32 yaw_cmd

float64 x_pos
float64 y_pos
float64 z_pos
float64 x_orient
float64 y_orient
float64 z_orient
float64 w_orient

float32 vel_x
float32 vel_y
float32 vel_z
float32 angular_vel_z

float32 min_threshold
float32 r_0
float32 d_0


